### PR TITLE
feat: initial implementation for code editor toolbar

### LINKED ISSUE #13: Implementation: Finalize Code Editor Toolbar
### Description
Create responsive toolbar with needed buttons for code editor. Button functionality is not needed.

### Acceptance Criteria
- Responsive toolbar is made for Code Editor 
- All needed buttons and fields are in the toolbar

### PR DESCRIPTION
**Pull request by yu-emily**
Based on issue #13 [Implementation: Finalize Code Editor Toolbar](https://github.com/The-Programming-Foundation/tutorials/issues/13)
Build responsive toolbar with buttons and fields based on this [UI design](https://user-images.githubusercontent.com/49451794/102140127-ae081c80-3e13-11eb-8940-65f061d94f86.png). Functionalities not yet completely implemented.

Libraries used:
bootstrap, [react-bootstrap](https://react-bootstrap.github.io/), [gatsby-plugin-google-fonts](https://www.gatsbyjs.com/plugins/gatsby-plugin-google-fonts/), [react-icons](https://react-icons.github.io/react-icons)

### CODE REVIEW

**Comment by yu-emily** [comment]
Currently, only the theme changes when you click on the sun/moon icon.
The demo can be viewed in Programming > Python > Hello Earth.
(Component currently uses useState. Will update to useRef when implementing datastore for editor.)

**Comment by ReesMcD** [comment]
There is a ticket to research CSS Libraries and Frameworks to use, so that we can unify around one package (maybe two if needed) so we might end up refactoring from the bootstrap solutions you implemented but I think as of right now it's good.

**Review by ReesMcD — APPROVED** [review]

File: src/components/EditorToolbar.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        fontWeights: {
+            body: 400,
+            bold: 700
+        },
+        fontSizes: [14, 16, 24, 32, 48, 64, 72],
+        width: [150, 200, 250, 300, 350, 400, 450, 500],
+        spacing: [25, 35, 45]
+    }
+
+    const buttonStyle = {
```
Comment (ashwinkjoseph) [inline src/components/EditorToolbar.jsx:35]: I see that we are using the native react styling and using it directly on HTML Elements. Didn't we agree on either using individual SCSS files or using Styled components?

Could you look into moving this into Styled components?

**Review by ashwinkjoseph — CHANGES_REQUESTED** [review]
Can we make use of styled-components rather than native styling?

Also, can you add some method of obtaining the code from the editor?

**Pull request closed without merging**